editor.planx.uk/src/ui/CollapsibleInput.stories.tsx

```diff
@@ -1,27 +1,35 @@
 import FeedbackInput from "@planx/components/shared/FeedbackInput";
-import { Meta, StoryFn, StoryObj } from "@storybook/react";
+import { Meta, StoryObj } from "@storybook/react";
 import React, { useState } from "react";
 
 import CollapsibleInput, { Props } from "./CollapsibleInput";
 
-const metadata: Meta = {
-  title: "Design System/Molecules/CollapsibleInput",
+const meta = {
+  title: "Design System/Atoms/Form Elements/CollapsibleInput",
   component: CollapsibleInput,
-};
+} satisfies Meta<typeof CollapsibleInput>;
 
-export const Basic: StoryObj<Props> = {
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+// TODO clarify use of args.children versus render here
```

Comment:
Yeah I also had this question whilst reviewing - it looks like `render()` is actually optional and will be defaulted using these `args` and the predefined `meta` right?